_includes/supporting_opposing_committees.html

```diff
@@ -1,24 +1,58 @@
+{% comment %}
+data:
+  committees = [
+    {
+      amount
+      filer_id || id
+      name
+    }
+  ]
+{% endcomment %}
+
 {% comment %}
 First we calculate the maximum for the money bar.
 {% endcomment %}
 {% assign max = 0 %}
 {% for committee in include.committees %}
-  {% assign max = max | plus: committee.amount %}
+  {% assign amount = committee.Total | default: committee.amount %}
+  {% assign max = max | plus: amount %}
 {% endfor %}
 
 {% for committee in include.committees %}
   {% comment %}
-  We're dealing with two different object shapes, a committee from collection, or
-  a committee from the referendum supporting/opposing organzation list. One has
-  `filer_id`, the other `id`.
+  We're dealing with three different object shapes
+    1. a committee from collection
+    and one of the following:
+    2. a committee from the referendum supporting/opposing organization list.
+    3. a committee from the candidate support_list/opposition_list organization list
+  committee collection has:
+    `filer_id`
+  referendum support/oppose org list has:
+    the other `id`
+  candidate support/oppose org list has:
+    `Filer_ID`
+    `Total`
+    `Filer_NamL`
   {% endcomment %}
-  {% assign filer_id = committee.filer_id | default: committee.id %}
 
-  <div><a href="{{ site.baseurl }}/committee/{{ filer_id }}/">{{ committee.name }}</a></div>
-  {% if committee.amount > 0 %}
-    <div>{{ committee.amount | dollars }}</div>
-    {% include money-bar.html value=committee.amount color=include.color max=max %}
+  {% if committee.id %}
+    {% assign filer_id = committee.id %}
+  {% elsif committee.Filer_ID %}
+    {% assign filer_id = committee.Filer_ID %}
+  {% else %}
+    {% assign filer_id = committee.filer_id %}
+  {% endif %}
+  {% assign filer_id = filer_id | downcase %}
+  {% assign money = committee.Total | default: committee.amount %}
+  {% assign name = committee.Filer_NamL | default: committee.name %}
+
+  {% if filer_id != "pending" %}
```

Comment:
s/pending/PENDING/

Reply:
No sure what you mean here
Instead of downcasing the filer_id [it now does](https://github.com/caciviclab/odca-jekyll/pull/271/commits/c5a8968dc40f344f08346b76812622dfe6696d71)

if filer_id != "pending" and filer_id != "Pending" and filer_id != "PENDING"

Is `s/pending/PENDING/` another solution?